### DISCUSSION TITLE
Gestione esami di stato

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#523] [Idee e proposte]
Buonasera ,come procede la richiesta fatta un anno fa circa l'integrazione della gestione degli esami di stato?Ho visto che anche in lampschool esiste ormai da anni questa funzione.

**Comment by trinko** [comment]
La gestione dell'esame non è prevista, perché viene gestito dall'applicazione MIUR Commissione Web.

Potrebbe essere utile un'integrazione con SIDI per l'invio dei dati degli studenti, ma non è una priorità.

**Reply by sempliciniluigi89** [reply]
lo chiedevo dal momento che in Argo esiste appieno la gestione degli esami  nonostante l'applicazione miur ,stesso dicasi per axios,classeviva,lampschool .Sinceramente giuaschool non mi sembra da meno.Si potrebbe proprio creare un collegamento a partire dagli scrutini delle classi quinte.

**Reply by trinko** [reply]
Il tempo a disposizione per lo sviluppo software è sempre troppo poco... Si deve per forza andare per priorità.
